aspnet/tutorials/first-web-api.rst

```diff
@@ -27,29 +27,25 @@ GET /api/todo          Get all to-do items       None          Array of to-do it
 GET /api/todo/{id}     Get an item by ID         None          To-do item
 POST /api/todo         Add a new item            To-do item    To-do item
 PUT /api/todo/{id}     Update an existing item   To-do item    None
+PATCH /api/todo/{id}   Update an existing item   To-do item    None
 DELETE /api/todo/{id}  Delete an item.           None          None
 =====================  ========================  ============  =============
 
 The following diagram show the basic design of the app.
 
 .. image:: first-web-api/_static/architecture.png
 
-- The client is whatever consumes the web API (browser, mobile app, and so forth). We aren’t writing a client in this tutorial.
+- The client is whatever consumes the web API (browser, mobile app, and so forth). We aren’t writing a client in this tutorial. We'll use `Postman <https://www.getpostman.com/>`__ or `Fiddler <http://www.fiddler2.com/fiddler2/>`__  for the client to test the app.
 - A *model* is an object that represents the data in your application. In this case, the only model is a to-do item. Models are represented as simple C# classes (POCOs).
 - A *controller* is an object that handles HTTP requests and creates the HTTP response. This app will have a single controller.
-- To keep the tutorial simple the app doesn’t use a database. Instead, it just keeps to-do items in memory. But we’ll still include a (trivial) data access layer, to illustrate the separation between the web API and the data layer. For a tutorial that uses a database, see :doc:`first-mvc-app/index`.
-
-Install Fiddler
----------------
-
-We're not building a client, we'll use `Fiddler <http://www.fiddler2.com/fiddler2/>`__ to test the API. Fiddler is a web debugging tool that lets you compose HTTP requests and view the raw HTTP responses.
+- To keep the tutorial simple, the app doesn’t use a database. Instead, it just keeps to-do items in memory. But we’ll still include a (trivial) data access layer, to illustrate the separation between the web API and the data layer. For a tutorial that uses a database, see :doc:`first-mvc-app/index`.
 
 Create the project
 ------------------
 
 Start Visual Studio. From the **File** menu, select **New** > **Project**.
 
-Select the **ASP.NET Core Web Application** project template. Name the project ``TodoApi`` and tap **OK**.
+Select the **ASP.NET Core Web Application (.NET Core)** project template. Name the project ``TodoApi`` and tap **OK**.
```

Comment:
In the following step the dialog screenshot shows Host in the cloud cleared but the steps don't say to clear it.  It's selected by default.